freeipa.spec.in

```diff
@@ -148,7 +148,8 @@ BuildRequires:  pki-base-python2
 BuildRequires:  python-pytest-multihost
 BuildRequires:  python-pytest-sourceorder
 BuildRequires:  python-jwcrypto
-BuildRequires:  python-custodia
+# 0.3: sd-notify and ipaserver.secrets.service
```

Comment:
Counter nit pick: These are the new features provided by Custodia 0.3: fully working sd-notify support and the ability to provide ```ipaserver.secerts.service``` with custom argsparser.